libs/sdk-ui-kit/src/syntaxHighlightingInput/tests/SyntaxHighlightingInput.test.tsx

```diff
@@ -67,29 +68,33 @@ describe("SyntaxHighlightingInput", () => {
             value: "this is a text content",
             className: "this-is-a-classname",
         };
-        const wrapper = renderComponent(props);
-        const value = wrapper.find("textarea").text();
+        renderComponent(props);
 
-        expect(value).toEqual(props.value);
-        expect(wrapper.find(`textarea.${props.className}`)).toHaveLength(1);
+        expect(screen.getByText("this is a text content")).toBeInTheDocument();
+        expect(document.querySelector(`textarea.${props.className}`)).toBeInTheDocument();
     });
 
     it("should call onChangeHandler function on value change", async () => {
         const onChange = jest.fn();
         const newValue = "new text content";
-        const wrapper = renderComponent({ onChange });
+        renderComponent({ onChange });
 
-        wrapper.find("textarea").simulate("change", { target: { value: newValue } });
-        expect(onChange).toHaveBeenCalledTimes(1);
+        fireEvent.change(screen.getByRole("textbox"), { target: { value: newValue } });
```

Comment:
we could use `userEvent` instead of `fireEvent`

Reply:
userEvent triggers the event every time it types. Fire event is more "accurate" for this test case.